### PR TITLE
Add suggestion when error happend by using TFLiteConverter.from_keras_model_file

### PR DESCRIPTION
**Pull request by theangels**
If the model has it own loss function, the processing of the transforming will failed. So I add the hint to let them try to use TFLiteConverter.from_session instead of using TFLiteConverter.from_keras_model_file. This hint was suggested by Googler in my previous PR (rejected), so I want to share it.And this PR will not changes the behavior of the converter for every user.

@wangtz

### CODE REVIEW

File: tensorflow/lite/python/lite.py [hunk 1]
```diff
@@ -597,23 +597,28 @@ def from_keras_model_file(cls,
     """
     _keras.backend.clear_session()
     _keras.backend.set_learning_phase(False)
-    keras_model = _keras.models.load_model(model_file)
-    sess = _keras.backend.get_session()
+    
+    try:
```
Comment (gargn) [inline tensorflow/lite/python/lite.py]: Please make the following changes:
1. Put the exception only around `keras_model = _keras.models.load_model(model_file)`
2. Make sure no line has over 80 characters
3. Use a more specific exception than "Exception"
4. Add a test case

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/lite/python/lite.py (hunk 1 above)
Comment (gargn) [reply tensorflow/lite/python/lite.py]: After thinking about it more, it is not guaranteed that the reason `load_model` fails is because custom parameters are allowed. If someone passes in an invalid file, then `load_model` fails as well. Please make sure to retain and print out the original message first and then add a suggestion to the user that it might be an issue with custom objects.

Comment (theangels) [reply tensorflow/lite/python/lite.py]: ok, I'll follow your suggestion.

**Comment by gargn** [comment]
After further thought, we have resolved the primary source of the issue - with adding support for `custom_objects` parameter. I think additionally adding support for `compile` parameter flag would be the preferred approach over an additional try/except.

**Comment by theangels** [comment]
OK, I see. The conventor v2 skip the load keras model step, maybe it can avoid the loading function errors.
